server/routes/prisons/sessionTemplates/addSessionTemplateController.ts

```diff
@@ -94,16 +97,33 @@ export default class AddSessionTemplateController {
       }),
       body('weeklyFrequency')
         .trim()
+        .toInt()
         .isInt({ min: 1, max: 12 })
         .withMessage('Enter a weekly frequency value between 1 and 12'),
       body(['validFromDateDay', 'validFromDateMonth', 'validFromDateYear'])
         .trim()
         .isInt({ min: 1 })
         .withMessage('Enter the date in number format for template start date'),
+      body(['validFromDateDay', 'validFromDateMonth'])
+        .trim()
+        .isInt()
+        .isLength({ min: 2, max: 2 })
```

Comment:
Wondering here if it makes more sense to _convert_ to integer and then check min/max is, say, 1/12 or 1/31. Then, when constructing the `Date`, just pad with zeros if necessary.

This means it would be more discerning (i.e. not let through `00` or `99`) and be more forgiving and accept input like `1 / 2 / 2023`.

I'll have a quick look to see if this would work....

Reply:
to int on the validation means the form values returns an integer, so will strip leading 0's (mentioned in another comment) 
had looked at converting the input at the stage we build the session template instead of validating, guess a decision needs to be made as to what makes more sense